src/matrices/linmin.c

```diff
@@ -93,12 +94,12 @@ double linmin(double *converged_f, double *converged_df,
 	  }
 
 	  if (*task != 'C') {  /* not converged; warning or error */
-	       if (verbose || *task == 'E')
+	       if ((verbose || mpb_verbosity >= 2 || *task == 'E')  && mpb_verbosity >= 2)
```

Comment:
Shouldn't this have been `mpb_verbosity >= 3`? Same for line 102.
Right now, this gets spammed multiple times for every k-point when calling mpb, which seems undesirable as default behavior.